health-services/project-factory/migration/main/V20240826145500__add_resource_and_campaign.sql

```diff
@@ -0,0 +1,92 @@
+CREATE TABLE eg_cm_campaign_details
+(
+    id character varying(128) NOT NULL,
+    tenantid character varying(64) NOT NULL,
+    status character varying(128) NOT NULL,
+    action character varying(64) NOT NULL,
+    campaignnumber character varying(128) NOT NULL,
+    hierarchytype character varying(128),
+    boundarycode character varying(64),
+    projectid character varying(128),
+    createdby character varying(128) NOT NULL,
+    lastmodifiedby character varying(128),
+    createdtime bigint NOT NULL,
+    lastmodifiedtime bigint,
+    additionaldetails jsonb,
+    campaigndetails jsonb,
+    campaignname character varying(250),
+    projecttype character varying(128),
```

Comment:
I need some reasoning on why all these fields were added, we will discuss in person tomorrow